tests/extmod/ujson_dumps.py

```diff
@@ -26,3 +26,5 @@
 print(json.dumps({"a":(2,[3,None])}))
 print(json.dumps('"quoted"'))
 print(json.dumps('space\n\r\tspace'))
+print(json.dumps({1:2, 3:4, 5.5:6}))
```

Comment:
Please just put one element in the dict, otherwise the test may fail if the order is different when printing.

Also, float tests need to go in the separate file `ujson_dumps_float.py`.

Reply:
Understood why this is important (undefined key order), but doesn't that mean we're missing coverage of multi-key objects?

In any case, I'll change it up.

Reply:
Updated

Reply:
> but doesn't that mean we're missing coverage of multi-key objects?

True.  A way to fix that would be to sort the resulting string, eg `print(sorted(json.dumps({1:2, 3:4})))`, but that's no an ideal solution.